### ISSUE TITLE
TypeError: Invalid locator at Function.webdriver.Locator.checkLocator 

### ISSUE THREAD

**Issue by mohammedshakirulla** [angular/protractor#988]
Hi,

I get the following error "TypeError: Invalid locator at Function.webdriver.Locator.checkLocator" when I run protractor tests on windows platform
- The error is observed on using "protractor.By.binding" locator
## \* Works fine when executed on Linux machine

Selenium server version - 2.42.2
Protractor version - 1.0.0-rcl
node.js version - 0.10.29

Error log
(C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\locators.js:243:9) at webdriver.WebElement.findElement (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\webdriver.js:1691:31) at C:\Users\UserDirectory\node_modules\e2e\Program_Name.js:71:36 at C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\goog\base.js:1243:15 at webdriver.promise.ControlFlow.runInNewFrame_ (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:1539:20) at notify (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:362:12) at notifyAll (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:331:7) at resolve (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:309:7) at fulfill (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\webdriver\promise.js:429:5) at C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\goog\base.js:1243:15 ==== async task ==== Asynchronous test function: it() at null. (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd\index.js:93:33) at null. (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\async-callback.js:45:37) at jasmine.Block.execute (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:1174:17) at jasmine.Queue.next_ (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:2209:31) at null._onTimeout (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:2199:18) at Timer.listOnTimeout [as ontimeout](timers.js:110:15)==== async task ==== Error at null. (C:\Users\UserDirectory\node_modules\e2e\Program_Name.js:57:5) at jasmine.Env.describe_ (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:913:21) at jasmine.Env.describe (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:898:15) at describe (C:\Users\UserDirectory\AppData\Roaming\npm\node_modules\protractor\node_modules\minijasminenode\lib\jasmine-1.3.1.js:658:27) at Object. (C:\Users\UserDirectory\node_modules\e2e\Program_Name.js:8:1) at Module._compile (module.js:456:26) at Object.Module._extensions..js (module.js:474:10) at Module.load (module.js:356:32) at Function.Module._load (module.js:312:12)

**Comment by netspend-staylor1** [comment]
LIke @kbaltrinic I copied the code from the protractor-cucumber module. All of a sudden, yesterday I started getting this message. I still cannot figure out what changed that caused this to start happening, but changing my require fixed it. Before, I required my project's own selenium-webdriver module:

webdriver = require('selenium-webdriver');

I changed it to:

webdriver = require(“../../node_modules/protractor/node_modules/selenium-webdriver”);

I then removed selenium-webdriver from the dependencies for my project.
